libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -600,7 +600,330 @@ async def _completion_with_retry_inner(
 
 
 class ChatVertexAI(_VertexAICommon, BaseChatModel):
-    """`Vertex AI` Chat large language models API."""
+    """Google Cloud Vertex AI chat model integration.
+
+    Setup:
+        You must have the langchain-google-vertexai Python package installed
+        .. code-block:: bash
+
+            pip install -U langchain-google-vertexai
+
+        And either:
+            - Have credentials configured for your environment (gcloud, workload identity, etc...)
+            - Store the path to a service account JSON file as the GOOGLE_APPLICATION_CREDENTIALS environment variable
+
+        This codebase uses the google.auth library which first looks for the application
+        credentials variable mentioned above, and then looks for system-level auth.
+
+        For more information, see:
+        https://cloud.google.com/docs/authentication/application-default-credentials#GAC
+        and https://googleapis.dev/python/google-auth/latest/reference/google.auth.html#module-google.auth.
+
+    Key init args — completion params:
```

Comment:
nits: maybe we can use a hint that these args might be overriden by the invoke kwargs?